core/pom.xml

```diff
@@ -0,0 +1,107 @@
+    </parent>
+    <groupId>io.stargate.core</groupId>
+    <artifactId>core</artifactId>
+    <properties>
+        <dropwizard.version>2.0.10</dropwizard.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.30</version>
+        </dependency>
+        <dependency>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+            <version>4.1.9</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
+            <version>${dropwizard.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.3</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.2.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>5.6.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.14.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>3.5.10</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <unpackBundle>true</unpackBundle>
+                    <instructions>
+                        <Bundle-Name>Core</Bundle-Name>
+                        <Bundle-Description>Core API for Stargate project</Bundle-Description>
+                        <Bundle-SymbolicName>io.stargate.core</Bundle-SymbolicName>
+                        <Bundle-Activator>io.stargate.core.CoreActivator</Bundle-Activator>
+                        <Import-Package><![CDATA[
+                          com.codahale.metrics,
+                          org.osgi.framework,
+                        ]]></Import-Package>
+                        <Export-Package><![CDATA[
+                          io.stargate.core.*,
```

Comment:
do we need to export whole `io.stargate.core.*,`? maybe the metrics `package` is enough?

Reply:
This makes the whole package to be available to be use by other bundles.
For now, only a single interface will be exported.